nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -547,6 +558,32 @@ public void shutdown() throws Exception {
         }
     }
 
+    void rejectPendingRequests() {
+        HttpRequestContainer container;
+        while ((container = getNextContainer()) != null) {
+            try {
+                HttpServletResponse response = container.getResponse();
```

Comment:
Could you please add a warning log message here as in case of the other `sendError()` calls.